packages/core/strapi/lib/services/entity-service/index.d.ts

```diff
@@ -73,11 +79,14 @@ export interface EntityService {
   ): Promise<T>;
 
   count<K extends keyof AllTypes, T extends AllTypes[K]>(uid: K, params: Params<T>): Promise<any>;
-  create<K extends keyof AllTypes, T extends AllTypes[K]>(uid: K, params: Params<T>): Promise<any>;
-  update<K extends keyof AllTypes, T extends AllTypes[K]>(
-    uid: K,
+  create<TSchemaUID extends Common.UID.ContentType>(
```

Comment:
Let's wait before updating those types (they're part of other tasks)

(even though I guess it was for testing purpose, let's try to keep the changes atomic for now)

Reply:
But the task in Jira says: `When accessing the create or update methods then I want types the data parameter to be accessible`, if we don't update this, they won't be accessible